conf/yarn.conf.template

```diff
@@ -10,7 +10,7 @@ gearpump {
       name = "Application Master"
       main = "org.apache.gearpump.experiments.yarn.master.YarnApplicationMaster"
       command = "$JAVA_HOME/bin/java"
-      memory = "1024"
+      memory = "2048"
```

Comment:
i originally had something similar to java command line where it could be "1024m" or "1g" - would that be preferable?